ddtrace/settings/config.py

```diff
@@ -223,7 +224,7 @@ def __init__(self):
         self._trace_compute_stats = asbool(
             os.getenv("DD_TRACE_COMPUTE_STATS", os.getenv("DD_TRACE_STATS_COMPUTATION_ENABLED", False))
         )
-        self._appsec_enabled = asbool(os.getenv("DD_APPSEC_ENABLED", False))
+        self._appsec_enabled = asbool(os.getenv(APPSEC_ENV, False))
```

Comment:
maybe worth using `envier` now that it is a dependency?

Reply:
It's interesting `envier` :clap: I think this refactor it could be better in other PR, what do you think?